h3py/api/old.py

```diff
@@ -0,0 +1,60 @@
+from .set_str import (
```

Comment:
It seems like this is an API more consistent with the C API - why is this the "old" API? Do we want to have a single API or multiple APIs? Which one should use a user of the library use?

Reply:
I just wanted to show that it was easy to make an API which conforms to the existing `h3-py` API. The term "old" just reveals my bias as towards which API I like better ;)

We were hoping to have multiple APIs, but with one default API which would probably be under the `h3.*` top-level namespace. Other APIs might live under something like `h3.api.*`.

Reply:
I've updated the naming to match the current python implementation.